### PR TITLE
:bug: Disable TLS 1.0 and TLS 1.1

### PR DESCRIPTION
**Pull request by as20203**
We found out that TLS 1.0 and TLS 1.1 are enabled despite being insecure. These changes are backported to release-1.3 branch, but without the configuration flags. Min version is set to TLS12 and Max version set to TLS13, with no config to change those.

### CODE REVIEW

**Comment by as20203** [comment]
@tuminoid do we allow setting cipher suites using the cipher suite flag for the hardcoded version release 1.3?

**Comment by tuminoid** [comment]
> @tuminoid do we allow setting cipher suites using the cipher suite flag for the hardcoded version release 1.3?

No configuration flags are added here. Just apply the default TLS config.

**Comment by tuminoid** [comment]
/retitle :bug: Disable TLS 1.0 and TLS 1.1

File: main.go [hunk 1]
```diff
@@ -225,6 +253,13 @@ func initFlags(fs *pflag.FlagSet) {
 		":9440",
 		"The address the health endpoint binds to.",
 	)
+	tlsCipherPreferredValues := cliflag.PreferredTLSCipherNames()
+	tlsCipherInsecureValues := cliflag.InsecureTLSCipherNames()
+	fs.StringVar(&tlsOptions.TLSCipherSuites, "tls-cipher-suites", "",
```
Comment (tuminoid) [inline main.go]: No flags whatsoever should be added in this PR. We only want to disable TLS 1.0 and TLS 1.1. Cipher suites are staying as default like they were before.

File: main.go [hunk 2]
```diff
@@ -376,3 +411,75 @@ func setupWebhooks(mgr ctrl.Manager) {
 		os.Exit(1)
 	}
 }
+
+// GetTLSOptionOverrideFuncs returns a list of TLS configuration overrides to be used
+// by the webhook server.
+func GetTLSOptionOverrideFuncs(options TLSOptions) ([]func(*tls.Config), error) {
```
Comment (tuminoid) [inline main.go:411]: We can clean up this function a lot, or maybe even remove it, since there is no options to check for.

File: main_test.go
Comment (tuminoid) [inline main_test.go:1]: We probably don't have anything to test, since we don't provide any options in this PR.

Comment (as20203) [reply main_test.go:1]: Yeah, I've removed those tests since the values are hardcoded.

File: main.go (hunk 2 above)
Comment (as20203) [reply main.go]: I've cleaned up this function with only hardcoded tls values and also removed other code which is redundant now such as providing options to this function which are hardcoded inside it.

**Review by tuminoid — COMMENTED** [review]
/lgtm

**Pull request merged**